tgui/.prettierrc.yml

```diff
@@ -1,12 +1,15 @@
 arrowParens: always
-bracketSpacing: true
+breakLongMethodChains: true
 endOfLine: lf
+importFormatting: oneline
 jsxBracketSameLine: true
 jsxSingleQuote: false
-printWidth: 80
+offsetTernaryExpressions: true
+printWidth: 72
 proseWrap: preserve
 quoteProps: preserve
 semi: true
 singleQuote: true
 tabWidth: 2
 trailingComma: es5
+useTabs: true
```

Comment:
This would suck for code reviews on Github, because Github treats tabs as 8 spaces regardless of our editorconfig, and a lot of auto-generated javascript and existing javascript on the web (and here) already uses two spaces for tabs.

Reply:
tbf the existing javascript would ctrl+s into using tabs if you have vsc setup, and it is odd for us to have tabs in one end of the codebase and spaces in another for reasons not related to something like YML requiring spaces or similar